lib/crypto/crypto_compat.c

```diff
@@ -158,7 +158,7 @@ crypto_compat_RSA_generate_key()
 {
 	RSA * key;
 
-#if OPENSSL_VERSION_NUMBER < 0x10100000L
+#if OPENSSL_VERSION_NUMBER < 0x00908000L
```

Comment:
Might as well start with 0.9.8.  Is this why we've had deprecation warnings from OS X, or was that a matter of OpenSSL being completely deprecated rather than specific functions?